x-pack/plugins/security_solution/server/lib/detection_engine/rules/types.ts

```diff
@@ -165,9 +172,11 @@ export const isRuleStatusFindTypes = (
 export interface CreateRulesOptions {
   alertsClient: AlertsClient;
   anomalyThreshold: AnomalyThresholdOrUndefined;
+  author: AuthorOrUndefined;
```

Comment:
This shouldn't be `OrUndefined` since you define it on the way in.

Reply:
Thanks for pairing and going over this with me -- changed it for `UpdateRulesOptions` as well! 🙂